javascript/javascript1/week2/main.js

```diff
@@ -0,0 +1,106 @@
+//freeCodeCamp user:@TatjanaOcedova
+
+//Flight booking fullname
+function getFullname(firstname, surname, useFormalName, gender) {
+  if (firstname.length === 0 || surname.length === 0) {
+    return "Fullname is empty";
+  } else if (useFormalName == true) {
+    switch (gender) {
+      case "female":
+        return "Lady " + firstname + " " + surname;
+        break;
+      case "male":
+        return "Lord " + firstname + " " + surname;
+        break;
+      default:
+        return firstname + " " + surname;
+        break;
+    }
+  } else {
+    return firstname + " " + surname;
+  }
+}
+
+const fullname1 = getFullname("Tatjana", "Ocedova", true, "female");
+const fullname2 = getFullname("", "Ocedov", false, "male");
+
+console.log(fullname1);
+console.log(fullname2);
+
+//Event application
+const weekday = [
+  "Sunday",
+  "Monday",
+  "Tuesday",
+  "Wednesday",
+  "Thursday",
+  "Friday",
+  "Saturday",
+];
+function getEventWeekday(daysLeft) {
+  const todaysDate = new Date();
+  const eventDate = new Date(
+    todaysDate.getFullYear(),
+    todaysDate.getMonth(),
+    todaysDate.getDate() + daysLeft
+  );
+  return weekday[eventDate.getDay()];
+}
+
+console.log(getEventWeekday(9));
+console.log(getEventWeekday(2));
+
+//Weather wear
+function weatherTemperature(celsius) {
+  if (celsius <= 5) {
+    return "warm cloths, jacket and boots";
+  } else if (celsius <= 15) {
+    return "raincoat and sneakers";
+  } else if (celsius <= 25) {
+    return "t-shirt, jeans and sandals";
+  } else {
+    return "beach wear";
+  }
+}
+const clothsToWear = weatherTemperature(45);
+console.log(clothsToWear);
+
+//Student manager
+const class07Students = [];
```

Comment:
Good understanding of const. Can be a little tricky with arrays. But it is right.